corehq/apps/export/forms.py

```diff
@@ -436,6 +439,80 @@ def get_multimedia_task_kwargs(self, export, download_id):
         return kwargs
 
 
+class EmwfFilterFormExport(FilterFormESExportDownloadForm):
+    def __init__(self, domain_object, *args, **kwargs):
+        self.domain_object = domain_object
+        super(EmwfFilterFormExport, self).__init__(domain_object, *args, **kwargs)
+
+        # remove the three filters. Rendered directly in view
+        del (self.fields['type_or_group'])
+        del (self.fields['user_types'])
+        del (self.fields['group'])
+        self.helper.label_class = 'col-sm-3 col-md-2 col-lg-2'
+        self.helper.field_class = 'col-sm-9 col-md-8 col-lg-3'
+        self.helper.layout = Layout(
+            super(EmwfFilterFormExport, self).extra_fields
+        )
+
+    def _get_es_user_types(self, mobile_user_and_group_slugs):
+        """
+        Override to fetch params from url and not form_data
+        Return a list of elastic search user types (each item in the return list
+        is in corehq.pillows.utils.USER_TYPES) corresponding to the selected
+        export user types.
+        """
+        es_user_types = []
+        user_types = ExpandedMobileWorkerFilter.selected_user_types(mobile_user_and_group_slugs)
+        export_user_types = [BaseFilterExportDownloadForm._USER_TYPES_CHOICES[i][0] for i in user_types]
+        export_to_es_user_types_map = {
+            self._USER_MOBILE: [utils.MOBILE_USER_TYPE],
+            self._USER_DEMO: [utils.DEMO_USER_TYPE],
+            self._USER_UNKNOWN: [
+                utils.UNKNOWN_USER_TYPE, utils.SYSTEM_USER_TYPE, utils.WEB_USER_TYPE
+            ],
+            self._USER_SUPPLY: [utils.COMMCARE_SUPPLY_USER_TYPE]
+        }
+        for type_ in export_user_types:
+            es_user_types.extend(export_to_es_user_types_map[type_])
+        return es_user_types
+
+    def _get_group(self, mobile_user_and_group_slugs):
+        # Override to fetch params from url and not form_data
+        group_ids = CaseListFilter.selected_group_ids(mobile_user_and_group_slugs)
```

Comment:
This method is overridden in CaseListFilter.
Moreover, For clarity i think using CaseListFilter for case export changes and LocationRestrictedMobileWorkerFilter for form export changes would be correct since they are used for rendering the filters as well. 
Both are subclassed from ExpandedMobileWorkerFilter. Would update at the places where ExpandedMobileWorkerFilter is being used instead of the above 2.